packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -153,9 +153,9 @@ export async function processOutboundDocumentQueryResps({
       numberOfParallelExecutions: 20,
     });
 
-    const documentRetrievalRequests = createOutboundDocumentRetrievalReqs({
+    const documentRetrievalRequests = await createOutboundDocumentRetrievalReqs({
       requestId,
-      cxId,
+      patient,
```

Comment:
because of this change, the test in the `create-outbound-document-retrieval-req.test.ts` file is failing. 
Now, instead of passing a random cxId, I need to pass a patient. I tried using `makePatient()`, but that creates a random facility ID. So when this function tries to get the organization with a random ID, it fails cuz the facility doesn't exist. How should I go about fixing this test?

Reply:
Problem discussed on a call